release/release_tests.yaml

```diff
@@ -3471,107 +3380,397 @@
+  working_dir: benchmarks
   legacy:
-    test_name: dataset_shuffle_sort_1tb_small_instances
-    test_suite: dataset_test
+    test_name: many_actors
+    test_suite: benchmark_tests
+
+  frequency: nightly
+  team: core
+  cluster:
+    cluster_env: app_config.yaml
+    cluster_compute: distributed.yaml
+    cloud_id: cld_HSrCZdMCYDe1NmMCJhYRgQ4p
+
+  run:
+    timeout: 3600
+    script: python distributed/test_many_actors.py
+    wait_for_nodes:
+      num_nodes: 65
 
+    type: job
+    file_manager: job
   stable: false
 
+- name: k8s_many_actors_smoke_test
+  group: core-scalability-test
+  working_dir: benchmarks
+  legacy:
+    test_name: many_actors_smoke_test
+    test_suite: benchmark_tests
+
+  frequency: multi
+  team: core
+  cluster:
+    cluster_env: app_config.yaml
+    cluster_compute: distributed_smoke_test.yaml
+    cloud_id: cld_HSrCZdMCYDe1NmMCJhYRgQ4p
+
+  run:
+    timeout: 3600
+    script: SMOKE_TEST=1 python distributed/test_many_actors.py
+    wait_for_nodes:
+      num_nodes: 2
+
+    type: job
+    file_manager: job
+  stable: false
+
+- name: k8s_many_tasks
+  group: core-scalability-test
+  working_dir: benchmarks
+  legacy:
+    test_name: many_tasks
+    test_suite: benchmark_tests
+
   frequency: nightly
   team: core
   cluster:
-    cluster_env: shuffle/shuffle_app_config.yaml
-    cluster_compute: shuffle/datasets_large_scale_compute_small_instances.yaml
+    cluster_env: app_config.yaml
+    cluster_compute: distributed.yaml
+    cloud_id: cld_HSrCZdMCYDe1NmMCJhYRgQ4p
 
   run:
-    timeout: 7200
-    script: python dataset/sort.py --num-partitions=1000 --partition-size=1e9
+    timeout: 3600
+    script: python distributed/test_many_tasks.py --num-tasks=10000
     wait_for_nodes:
-      num_nodes: 20
-      timeout: 900
-    type: sdk_command
-    file_manager: sdk
+      num_nodes: 65
 
-- name: dataset_shuffle_push_based_random_shuffle_1tb
-  group: core-dataset-tests
-  working_dir: nightly_tests
+    type: job
+    file_manager: job
+
+  smoke_test:
+    frequency: multi
+    cluster:
+      cluster_env: app_config.yaml
+      cluster_compute: distributed_smoke_test.yaml
+    run:
+      timeout: 3600
+      script: python distributed/test_many_tasks.py --num-tasks=100
+      wait_for_nodes:
+        num_nodes: 2
+
+      type: job
+      file_manager: job
+  stable: false
+
+- name: k8s_many_pgs
+  group: core-scalability-test
+  working_dir: benchmarks
   legacy:
-    test_name: dataset_shuffle_push_based_random_shuffle_1tb
-    test_suite: dataset_test
+    test_name: many_pgs
+    test_suite: benchmark_tests
+
+  frequency: nightly
+  team: core
+  cluster:
+    cluster_env: app_config.yaml
+    cluster_compute: distributed.yaml
+    cloud_id: cld_HSrCZdMCYDe1NmMCJhYRgQ4p
+
+  run:
+    timeout: 3600
+    script: python distributed/test_many_pgs.py
+    wait_for_nodes:
+      num_nodes: 65
+
+    type: job
+    file_manager: job
+  stable: false
+
+- name: k8s_many_pgs_smoke_test
```

Comment:
?

Reply:
Hmm idk how these are added back lol... Let me fix it

Reply:
Fixed